readthedocs/builds/version_slug.py

```diff
@@ -28,6 +28,15 @@
 from builtins import range
 
 
+def get_fields_with_model(cls):
+    return [
```

Comment:
If this is a prescribed Django fix for this, perhaps link to the permalink in docs here to explain what this is for.